docs/quick-start/create-a-connection-string.txt

```diff
@@ -0,0 +1,63 @@
+.. _laravel-quick-start-connection-string:
+
+==========================
+Create a Connection String
+==========================
+
+You can connect to your MongoDB deployment by providing a
+**connection URI**, also called a *connection string*, which
+instructs the driver on how to connect to a MongoDB deployment
+and how to behave while connected.
+
+The connection string includes the hostname or IP address and
+port of your deployment, the authentication mechanism, user credentials
+when applicable, and connection options.
+
+To connect to an instance or deployment not hosted on Atlas, see the
+:manual:`Connection Strings </reference/connection-string/>` in the Server
+manual.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Find your MongoDB Atlas Connection String
+
+      To retrieve your connection string for the deployment that
+      you created in the :ref:`previous step <laravel-quick-start-create-deployment>`,
+      log in to your Atlas account and navigate to the
+      :guilabel:`Database` section and click the :guilabel:`Connect` button
+      for your new deployment.
```

Comment:
S: I think it would be helpful to break this sentence up

```suggestion
      log in to your Atlas account. Then, navigate to the
      :guilabel:`Database` section and click the :guilabel:`Connect` button
      for your new deployment.
```